README.md

```diff
@@ -391,25 +391,6 @@ rescue JWT::InvalidIatError
 end
 ```
 
-**Adding Leeway**
```

Comment:
A comment should be provided that this feature has been remove in version 2.3.0

Reply:
Updated the docs - let me know if that looks correct.